packages/frontend/src/document/analysis_document_editor.tsx

```diff
@@ -31,6 +31,9 @@ import type { AnalysisDocument, ModelDocument } from "./types";
 
 import PanelRight from "lucide-solid/icons/panel-right";
 import PanelRightClose from "lucide-solid/icons/panel-right-close";
+import { handleExportSVG } from "../visualization";
+import { Download } from "lucide-solid";
+import { Tooltip } from "@kobalte/core/tooltip";
```

Comment:
The `Tooltip` component is not being used here (and the linter should report this, except that the merge conflict is preventing the CI from running).